openstack_controller/tests/conftest.py

```diff
@@ -138,9 +138,16 @@ def mock_responses(microversion_headers):
 
     def process_files(dir, response_parent):
         for file in dir.rglob('*'):
-            if file.is_file() and file.stem != ".slash":
+            if file.is_file():
```

Comment:
As we've talked this shouldn't be changed

Reply:
Done.